content/zh/docs/ambient/architecture/traffic-redirection/index.md

```diff
@@ -160,10 +160,10 @@ COMMIT
 COMMIT
 {{< /text >}}
 
-命令输出显示，额外的 Istio 特定链已被添加到应用程序 Pod
-网络命名空间内的 netfilter/iptables 中的 NAT 和 Mangle 表中。
-所有进入 Pod 的 TCP 流量都会被重定向到 ztunnel 代理进行入口处理。
-如果流量是明文（源端口不等于 15008），会被重定向到 in-Pod ztunnel 明文侦听端口 15006。
-如果流量是 HBONE（源端口等于 15008），会被重定向到 in-Pod ztunnel HBONE 侦听端口 15008。
-任何离开 Pod 的 TCP 流量都会被重定向到 ztunnel 的端口 15001 进行出口处理，
+命令输出显示，在应用程序 Pod 的网络命名空间内，
+netfilter/iptables 中的 NAT 和 Mangle 表中添加了其他 Istio 特定链。
```

Comment:
seems not a normal case of sentence to read in Chinese. A copy of Machine Translation?